src/helpers/pivot/spreadsheet_pivot/date_spreadsheet_pivot.ts

```diff
@@ -10,29 +10,33 @@ export function createDate(dimension: PivotDimension, value: FieldValue["value"]
   }
   if (!MAP_VALUE_DIMENSION_DATE[granularity].set.has(value)) {
```

Comment:
Shouldn't we early return `if(value === null) return null;` before this if ? Or is there an use to bother adding null to every granularity set, and to stringified null in the values?

Reply:
For now the `null` value should be returned as a string, but I could early return `"null"`

Reply:
> For now the null value should be returned as a string, but I could early return "null"

My bad, `createDate` should returns `null`. It's the caller of this function which stringify the value.
It's updated with your suggestion :)